src/server/logs/cmds/cmds.go

```diff
@@ -166,25 +192,31 @@ func Cmds(pachCtx *config.Context, pachctlCfg *pachctl.Config) []*cobra.Command
 			}
 			switch {
 			case cmd.Flag("logql").Changed:
-				if cmd.Flag("project").Changed || cmd.Flag("pipeline").Changed || cmd.Flag("datum").Changed || cmd.Flag("app").Changed {
+				if hasDisallowedFlags(cmd, "project", "pipeline", "datum", "app", "job") {
 					fmt.Fprintln(os.Stderr, "only one of [--logQL | --project PROJECT --pipeline PIPELINE | --datum DATUM] may be set")
 					os.Exit(1)
 				}
 				addLogQLRequest(req, logQL)
 			case cmd.Flag("pipeline").Changed:
-				if cmd.Flag("datum").Changed || cmd.Flag("app").Changed {
+				if hasDisallowedFlags(cmd, "datum", "app") {
 					fmt.Fprintln(os.Stderr, "only one of [--logQL | --project PROJECT --pipeline PIPELINE | --datum DATUM] may be set")
 					os.Exit(1)
 				}
 				addPipelineRequest(req, project, pipeline)
 			case cmd.Flag("project").Changed:
-				if cmd.Flag("datum").Changed || cmd.Flag("app").Changed {
+				if hasDisallowedFlags(cmd, "datum", "app") {
 					fmt.Fprintln(os.Stderr, "only one of [--logQL | --project PROJECT --pipeline PIPELINE | --datum DATUM] may be set")
 					os.Exit(1)
 				}
 				addProjectRequest(req, project)
+			case cmd.Flag("datum").Changed && cmd.Flag("job").Changed:
+				if hasDisallowedFlags(cmd, "app") {
+					fmt.Fprintln(os.Stderr, "only one of [--logQL | --project PROJECT --pipeline PIPELINE | --datum DATUM] may be set")
```

Comment:
Need to update the error message (which should probably be a `const` …) to `only one of [--logQL | --project PROJECT --pipeline PIPELINE | --datum DATUM [--job JOB]] may be set"`